examples/dsa_hisa/pool_mqa_fp8.py

```
@@ -0,0 +1,257 @@
+        weights,
+        cu_blocked_ks,
+        cu_blocked_ke,
+        block_N=block_N,
+    )
+    clean_and_maintain_logits_interface(got, cu_blocked_ks, cu_blocked_ke)
+
+    ref = ref_pool_mqa_fp8(q, blocked_k, blocked_k_scale, weights)
+    ref = ref_clean_and_maintain_logits(ref, cu_blocked_ks, cu_blocked_ke)
+
+    # After the mask, +/-inf positions must agree exactly. Compare the
+    # remaining finite values under an fp8×fp8 GEMM tolerance.
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)